File: SEP3_Client/Mediator/Information/AccountPackage.cs
using SEP3_Client.Model.List.UserList;
using SEP3_Client.Model.Unit.User;

namespace SEP3_Client.Mediator.Information
{
    public class AccountPackage : InformationPackage
    {
        public AccountList SendList { get; set; }

        public AccountPackage(AccountList accountList,string keyword) : base(InformationType.ACCOUNT,keyword)
        {
            SendList = accountList.Copy();
        }

        public AccountPackage(Account account,string keyword) : base(InformationType.ACCOUNT,keyword)
        {
            SendList = new AccountList();
            SendList.AddNewAccount(account);
        }

        public AccountList GetSendList() {
            return SendList;
        }
    }
}

File: SEP3_UserSystem/src/main/java/com/example/SEP3_UserSystem/mediator/information/AccountPackage.java
package com.example.SEP3_UserSystem.mediator.information;

import com.example.SEP3_UserSystem.model.domain.list.userList.AccountList;
import com.example.SEP3_UserSystem.model.domain.unit.user.Account;

public class AccountPackage extends InformationPackage
{
    private AccountList SendList;

    public AccountPackage(AccountList accountList,String keyword)
    {
        super(InformationType.ACCOUNT,keyword);
        SendList = accountList.copy();
    }

    public AccountPackage(Account account, String keyword)
    {
        super(InformationType.ACCOUNT,keyword);
        SendList = new AccountList();
        SendList.addNewAccount(account);
    }

    public AccountList getSendList() {
        return SendList;
    }
}
